common/src/main/scala/explore/model/reusability.scala

```diff
@@ -59,4 +60,5 @@ object reusability {
   implicit def modelUndoStacksReuse[F[_]]: Reusability[ModelUndoStacks[F]]           = Reusability.derive
   // Move to lucuma-ui
   implicit val semesterReuse: Reusability[Semester]                                  = Reusability.derive
+  implicit val cssReuse: Reusability[Css]                                            = Reusability.by(_.htmlClass)
```

Comment:
I was surprised I couldn't find a reuseability for Css anywhere. I can move this and the above to lucuma-ui and cut a release of it. (Unless I missed an instance somewhere)

Reply:
Maybe it should be in scalajs-react-common

Reply:
Actually, I did find an instance, but only in seqexec

Reply:
> Maybe it should be in scalajs-react-common

Oh, good point.